packages/components/src/components/avatar/avatar.component.ts

```diff
@@ -13,24 +13,19 @@ import type { AvatarType } from './avatar.types';
 import { getAvatarIconSize, getAvatarTextFontSize, getPresenceSize } from './avatar.utils';
 
 /**
- * The `mdc-avatar` component is used to represent a person or a space.
- * An avatar can be an icon, initials, counter and photo.
+ * The `mdc-avatar` component represents a person or a space. It displays as a photo, initials, icon, or counter.
  *
- * To set the photo of an avatar,
- * you need to set "src" attribute.
+ * ## Display priority (when multiple attributes are set)
```

Comment:
The heading should be short, also it is a good practice to put a short sentence before a list to describe what is. 

so do this instead:


```suggestion
 * ## Display priority 
 * 
 * Priorities of rendered content when multiple attributes set:
```